web-frontend/modules/core/components/RadioCard.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="radio-card" :class="classNames" @click="select(value)">
```

Comment:
I like that we have a reusable component for this one! It doesn't have to be in this merge request, but it would be great to have a storybook component for this one.